js/graphData.js

```diff
@@ -1,181 +1,149 @@
+      },
     },
 
     // right-hand y axis
     y2: {
       show: true, // add second y-axis
       label: {
-        text: "Watered Today (0 = No, 1 = Yes)",
-        position: 'outer-middle'
+        text: 'Watered Today (0 = No, 1 = Yes)',
+        position: 'outer-middle',
       },
       tick: {
         count: 2, // We don't need float tick values between a "yes" and "no" option
-        values: [0,1] // makes sure values aren't floating points
+        values: [0, 1], // makes sure values aren't floating points
       },
-    }
-  }
+    },
+  },
 });
 
-
-
 // Get data from database, parse into arrays
 async function getJSONData() {
-    var unix_time = [];
-    var date = [];
-    var soil_temp = [];
-    var soil_moisture = [];
-    var air_temp = [];
-    var air_humidity = [];
-    var heat_index = [];
-    var sunlight = [];
-    var watered_today = [];
-    
-    
-    date.push("x");
-    soil_temp.push("Soil Temp (°C)");
-    soil_moisture.push("Soil Moisture (%)");
-    air_temp.push("Air Temp (°C)");
-    air_humidity.push("Humidity (%)");
-    heat_index.push("Heat Index (°C)");
-    sunlight.push("Sunlight (%)");
-    watered_today.push("Watered Today (0/1)");
-    
-    
-    try {
-        $.get("../databaseconnect.php",
-        function (data) {
-            // alert("success");
-            
-            // for (var i in data) {
-            //     unix_time.push(data[i].unix_time);
-            //     date.push(data[i].date);
-            //     soil_temp.push(data[i].soil_temp);
-            //     soil_moisture.push(data[i].soil_moisture);
-            //     air_temp.push(data[i].air_temp);
-            //     air_humidity.push(data[i].air_humidity);
-            //     heat_index.push(data[i].heat_index);
-            //     sunlight.push(data[i].sunlight);
-            //     watered_today.push(data[i].watered_today);
-            // }
-    
-        })
-        .done(function(data) {
-            
-            for (var i in data) {
-                unix_time.push(data[i].unix_time);
-                date.push(data[i].date);
-                soil_temp.push(data[i].soil_temp);
-                soil_moisture.push(data[i].soil_moisture);
-                air_temp.push(data[i].air_temp);
-                air_humidity.push(data[i].air_humidity);
-                heat_index.push(data[i].heat_index);
-                sunlight.push(data[i].sunlight);
-                watered_today.push(data[i].watered_today);
-            }
-            // alert("Second success");
-        })
-        .fail(function() {
-            console.log("Error getting data");
-        });
-    
-    return {
-            unix_time: unix_time,
-            date: date,
-            soil_temp: soil_temp,
-            soil_moisture: soil_moisture,
-            air_temp: air_temp,
-            air_humidity: air_humidity,
-            heat_index: heat_index,
-            sunlight: sunlight,
-            watered_today: watered_today
-        };
```

Comment:
If the key has the same name as the value JavaScript will automatically make the key the variables name so:`{key_name: key_name}` can become: `{key_name}`